### ISSUE TITLE
Error on Reload if server not started

### ISSUE THREAD

**Issue by jayharris** [AveVlad/gulp-connect#16]
#### tldr

If the server has not started, yet, `connect.reload()` will fail on [index.js:60](https://github.com/AveVlad/gulp-connect/blob/master/index.js#L60), `opt.livereload`
#### detail

`TypeError: Cannot read property 'livereload' of undefined` occurs against [index.js:60](https://github.com/AveVlad/gulp-connect/blob/master/index.js#L60), attempting to read `opt.livereload`. However, if `connect.server()` has not been executed, yet, `opt` has not been initialized and remains undefined.

Using the example `gulpfile.coffee` from `README.md`, the script will work. However, if a `build` task is created that includes only `['html','styus']`, such as for a deployment script, then `gulp build` will fail on the `.pipe connect.reload()` commands.

``` coffee
gulp = require("gulp")
stylus = require("gulp-stylus")
connect = require("gulp-connect")

gulp.task "connect", connect.server(
  root: ['app']
  port: 1337
  livereload: true
  open:
    browser: "chrome" # if not working OS X browser: 'Google Chrome'
)

gulp.task "html", ->
  gulp.src("./app/*.html").pipe connect.reload()

gulp.task "stylus", ->
  gulp.src("./app/stylus/*.styl")
    .pipe(stylus())
    .pipe(gulp.dest("./app/css"))
    .pipe connect.reload()

gulp.task "watch", ->
  gulp
    .watch ["./app/*.html"], ["html"]
  gulp
    .watch ["./app/stylus/*.styl"], ["stylus"]

gulp.task "default", [
  "connect"
  "stylus"
  "watch"
]

gulp.task "build", [
  "html"
  "stylus"
]
```
